modules/violations-view-deployment-tab.adoc

```diff
@@ -11,34 +11,58 @@ The *Deployment* tab of the *Details* panel displays details of the deployment t
-* *Deployment name*: The name of the deployment.
-* *Deployment Type*: The type of the deployment.
-* *Cluster*: The name of the cluster where the container is deployed.
-* *Replicas*: The number of the replicated deployments.
-* *Namespace*: The unique identifier for the deployed cluster.
-* *Updated*: The time and date when the deployment was updated.
-* *Labels*: The labels that apply to the selected deployment.
-* *Annotations*: The annotations that apply to the selected deployment.
-* *Service Account*: The name of the service account for the selected deployment.
+The *Deployment overview* section lists the following information:
+
+* *Deployment ID*: The alphanumeric identifier for the deployment
+* *Deployment name*: The name of the deployment
+* *Deployment type*: The type of the deployment
+* *Cluster*: The name of the cluster where the container is deployed
+* *Namespace*: The unique identifier for the deployed cluster
+* *Replicas*: The number of the replicated deployments
+* *Created*: The time and date when the deployment was created
+* *Updated*: The time and date when the deployment was updated
+* *Labels*: The labels that apply to the selected deployment
+* *Annotations*: The annotations that apply to the selected deployment
+* *Service Account*: The name of the service account for the selected deployment
 
 [discrete]
 == Container configuration section
 
 The container configuration section lists the following information:
 
-* *Image Name*: The name of the image for the selected deployment.
-* *Resources*:
-** *CPU Request (cores)*: The number of cores requested by the container.
-** *Memory Request (MB)*: The memory size requested by the container.
-* *Volumes*:
+* *containers*: For each container, provides the following information:
+** *Image name*: The name of the image for the selected deployment. Click the name to view more information about the image.
+** *Resources*: This section provides information for the following fields:
+*** *CPU request (cores)*: The number of cores requested by the container.
+*** *CPU limit (cores)*: The maximum number of cores that can be requested by the container.
+*** *Memory request (MB)*: The memory size requested by the container.
+*** *Memory limit (MB)*: The maximum memory that can be requested by the container.
+** *volumes*: Volumes mounted in the container, if any.
+** *secrets*: Secrets associated with the selected deployment. For each secret, provides information for the following fields:
+*** *Name*: Name of the secret.
+*** *Container path*: Location where the secret is stored.
 ** *Name*: The name of the location where the service will be mounted.
 ** *Source*: The data source path.
 ** *Destination*: The path where the data is stored.
 ** *Type*: The type of the volume.
-* *Secrets*: Secrets associated with the selected deployment.
+
+[discrete]
+== Port configuration section
+
+Provides information about the ports in the deployment, including the following fields:
```

Comment:
Suggest sticking with a complete sentence for the first line, to match the other sections. E.g., if accurate re: markup:
```suggestion
The *Port configuration* section provides information about the ports in the deployment, including the following fields:
```